File: C#/BluffinMuffin.Protocol.DataTypes/Enums/BuyInParameterEnum.cs
using System.ComponentModel;

namespace BluffinMuffin.Protocol.DataTypes.Enums
{
    /// <summary>
    /// 
    /// </summary>
    public enum BuyInParameterEnum
    {
        /// <summary>
        /// Fixed Amount
        /// </summary>
        [Description("Fixed Amount")]
        FixedAmount,

        /// <summary>
        /// Multiplicator
        /// </summary>
        [Description("Multiplicator")]
        Multiplicator,

        /// <summary>
        /// Unlimited
        /// </summary>
        [Description("Unlimited")]
        Unlimited
    }
}

File: Python/bluffinmuffin/protocol/enums/buyin_parameter_enum.py
from enum import Enum


class BuyInParameterEnum(Enum):
    FixedAmount = 0
    Multiplicator = 1
    Unlimited = 2

    @classmethod
    def parse(cls, str):
        return cls[str]

    @classmethod
    def to_string(cls, value):
        return value.name
